src/store/classes/RequeteDeRecherche.ts

```diff
@@ -406,6 +407,12 @@ class RequeteDeRecherche extends VuexModule {
       });
       criteria.push(criterionRcr);
 
+      // Critère PPN
+      const criterionPpn = new CriterionPpn(this.globalPpnTypedInNumber);
```

Comment:
C'est this.getGlobalOptionsPpnSelected  qu'il faut lui passer en paramètre. ça correspond à l'Enum ET/OU/SAUF